src/test/scala/io/github/mkotsur/firebase/rest/FirebaseClientTest.scala

```diff
@@ -138,6 +160,22 @@ class FirebaseClientTest extends FunSpec with Matchers with ScalaFutures with Tr
           fc.get[Int](s"temp/pushed/$pushedChildName").futureValue shouldBe Some(43)
           fc.delete("temp").futureValue shouldBe ((): Unit)
         }
+        
+      it("should update the values of an existing list into Firebase") {
+        val adminCredential = AdminCredentials(validJsonKey)
+        implicit val token = FirebaseClient.getToken(adminCredential).success.value
+        val fc = new FirebaseClient(projectId)
+
+        fc.put[List[String]](List("apple", "cherry"), "listsEverywhere/listsOfFruits")
+        Thread.sleep(1000)
```

Comment:
For some reason I didn't think about it during the hackathon, but there is a very nice solution to this problem without hardcoding any numbers. And it will not need to vait the whole second if the value is ready sooner. It's [Eventually](http://doc.scalatest.org/2.2.0/index.html#org.scalatest.concurrent.Eventually).

Please note, that eventually block can also return a value, so you can do something like:

```
        val currentFruits = eventually {
          val result = fc.get[List[String]]("listsEverywhere/listsOfFruits").futureValue
          result shouldBe Some(List("apple", "cherry"))
          result
        }
```